src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -663,6 +663,24 @@ private void TakeIncrementalSnapshot(BoundNode node)
             _snapshotBuilderOpt?.TakeIncrementalSnapshot(node, State);
         }
 
+        private void SetUpdatedSymbol(BoundNode node, Symbol originalSymbol, Symbol updatedSymbol)
+        {
+            if (_updatedSymbolMapOpt is object)
+            {
+                var key = (node, originalSymbol);
+                if ((object)originalSymbol != updatedSymbol)
+                {
+                    _updatedSymbolMapOpt[key] = updatedSymbol;
+                }
+                else if (_updatedSymbolMapOpt.TryGetValue(key, out var currentSymbol) && (object)currentSymbol != updatedSymbol)
```

Comment:
>if (_updatedSymbolMapOpt.TryGetValue(key, out var currentSymbol) && (object)currentSymbol != updatedSymbol) [](start = 21, length = 107)

Is this check necessary?

Reply:
I just unconditionally remove I guess, I'll update in the next PR.